content/blog/how-to-create-a-ci-cd-pipeline-to-execute-automation-test-suite-with-hpe-greenlake-for-containers.md

```diff
@@ -0,0 +1,139 @@
+---
+title: How to Set Up an Automation Pipeline to Achieve Observability on HPE
+  GreenLake for Containers
+date: 2022-06-09T11:05:39.047Z
+featuredBlog: false
+priority: 1
+author: Sweta Katkoria
+authorimage: /img/swetakatkoria_photo.jpg
+thumbnailimage: /img/thumbnail2.jpg
+tags:
+  - hpe-greenlake
+  - Automation
+  - Grafana
+  - observability
+---
+## Introduction
+
+*An HPE GreenLake Cloud Service, HPE GreenLake for Containers, is built upon HPE Ezmeral Runtime Enterprise and runs on a container-based Kubernetes orchestrated infrastructure.* Using the Clusters module of the HPE GreenLake Central dashboard, one can perform several cluster-related operations. A cluster can be created using default or custom machine blueprints, deleted, scaled up, or scaled-down using this module. A user can navigate to the HPE Ezmeral Runtime through which various applications can be deployed and containerization can be achieved. Additionally, the cluster detail page provides some interesting insights like CPU usage, memory allocation, and storage availability. Metrics like these are crucial to tracking application usage and, subsequently, determining the resources required to grow the business. Here is the image mentioned above.
```

Comment:
Did you really want to italicize this whole sentence, or just "HPE GreenLake for Containers"?

In the 4th sentence, should the word "dashboard" be after "...can navigate to the HPE Ezmeral Runtime..."? I remember seeing that in another GreenLake blog post that talks of similar things.

Suggest changing the the last sentence here to read: "The image below illustrates what I mean."